### ISSUE TITLE
Crash on resize game window during duel

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#339]
```
Exception from controller.runGame: Destination component not connected to component tree hierarchy
java.lang.Error: Destination component not connected to component tree hierarchy
	at javax.swing.SwingUtilities.convertPoint(Unknown Source)
	at magic.ui.duel.ImageModeBattlefieldPanel.getLocationOnDuelPanel(ImageModeBattlefieldPanel.java:142)
	at magic.ui.duel.ImageModeBattlefieldPanel.setAnimationEvent(ImageModeBattlefieldPanel.java:135)
	at magic.ui.duel.DuelPanel.setAnimationEvent(DuelPanel.java:278)
	at magic.ui.SwingGameController.setAnimationEvent(SwingGameController.java:714)
	at magic.ui.SwingGameController.executeNextEventOrPhase(SwingGameController.java:721)
	at magic.ui.SwingGameController.runGame(SwingGameController.java:666)
	at magic.ui.duel.DuelPanel$2$1.run(DuelPanel.java:127)
```

**Comment by ShawnieBoy** [comment]
I was click-dragging the corner of the window, game with animations off

**Comment by lodici** [comment]
I have managed to replicate this intermittently by resizing the window by vigorously dragging the frame corner back and forth. Very difficult to replicate at will but my initial feeling is that it is trying to redraw and re-layout the game board multiple times per second for each resize event as the window is re-sized.

**Comment by ShawnieBoy** [comment]
:blush: No comments on my screen dragging methods...

It's only happened the once, I normally resize before starting a duel. Is it even possible for it to remember it's last screen size when closing and re-opening? The default size is a bit small for me. :grin:

**Comment by lodici** [comment]
I think the fix for #419 may have reduced the chances of this happening to negligible proportions.
